### PR TITLE
Improve `CCard` and `CCardDialog`

### PR DESCRIPTION
**Pull request by ItsPierre**
Improve responsiveness of `CCard`, add property `titleSize` for `CCard` and `CCardDialog` and add more stories for `CCard`

### CODE REVIEW

File: src/components/CCard.vue [hunk 1]
```diff
@@ -108,17 +113,48 @@ withDefaults(
...
   },
 );
+
+/**
+ * Computes the classes for the title component.
+ */
+const titleClass = computed<string>(() => {
+  let classes = "";
+
+  classes += "text-" + props.titlePosition;
```
Comment (johannes-huther) [inline src/components/CCard.vue]:
```suggestion
  let classes = "text-" + props.titlePosition;
```

File: src/components/CCard.vue [hunk 2]
```diff
@@ -108,17 +113,48 @@ withDefaults(
...
+  .v-card-title.large {
+    line-height: normal;
+
+    @media (max-width: map-get($grid-breakpoints, "md")) {
+      font-size: 26px !important;
+    }
+
+    @media (min-width: map-get($grid-breakpoints, "md")) {
+      font-size: 30px !important;
+    }
```
Comment (johannes-huther) [inline src/components/CCard.vue:155]: what happens exactly at the border?

File: stories/CCard.stories.ts [hunk 3]
```diff
@@ -62,6 +62,43 @@ export const Default: Story = {
   }),
 };
 
+export const TitlePosition: Story = {
```
Comment (johannes-huther) [inline stories/CCard.stories.ts]: rename `CenteredTitle`

File: stories/CCard.stories.ts [hunk 4]
```diff
@@ -62,6 +62,43 @@ export const Default: Story = {
...
+    Cards main content.
+</c-card>
+    `,
+  }),
+  parameters: createStorybookParameters({
+    slotTemplate: "Cards main content.",
+  }),
+};
+
+export const TitleSize: Story = {
```
Comment (johannes-huther) [inline stories/CCard.stories.ts]: Rename `LargeTitle`

**Review by johannes-huther — CHANGES_REQUESTED** [review]

File: src/components/CCard.vue (hunk 2 above)
Comment (ItsPierre) [reply src/components/CCard.vue:155]: both should be including the border, so any of these two will win

**Review by johannes-huther — APPROVED** [review]

**Pull request merged**